fairyfishgui.py

```diff
@@ -177,7 +178,7 @@ def square2idx(self, square):
     def render_square(key, location):
         square_color = SQUARE_COLORS[(location[0] + location[1]) % 2]
         font_size = min(sg.Window.get_screen_size()) // 50
-        button = sg.Button(size=(3, 2), button_color=square_color, pad=(0, 0), font='Any {}'.format(font_size), key=key)
+        button = sg.Button(size=(4, 2), button_color=square_color, pad=(0, 0), font='Any {}'.format(font_size), key=key)
```

Comment:
For me the layout looks better with the previous size. I think this depends on screen resolution, so it is a bit tricky to make it look good independent of screen resolution with this framework. I would suggest to split this topic out into a separate PR/issue as it seems rather independent.